neuralprophet/conformal.py

```diff
@@ -146,8 +170,12 @@ def plot(self, plotting_backend: str):
         """
         method = self.method.upper() if "cqr" in self.method.lower() else self.method.title()
         if plotting_backend == "plotly":
-            fig = plot_nonconformity_scores_plotly(self.noncon_scores, self.alpha, self.q_hat, method)
+            fig = plot_nonconformity_scores_plotly(self.noncon_scores, self.alpha, self.q_hats[0], method)
         elif plotting_backend == "matplotlib":
-            fig = plot_nonconformity_scores(self.noncon_scores, self.alpha, self.q_hat, method)
+            if self.n_forecasts == 1:
+                # includes nonconformity scores of the first timestep
+                fig = plot_nonconformity_scores(self.noncon_scores, self.alpha, self.q_hats[0], method)
```

Comment:
Ideally we would have the same behavior for `self.n_forecasts == 1` and other values of `n_forecasts`. 
Maybe we could instead have the special plot that includes the scores be a separate plotting utility call instead of automatically overwriting the standard plotting utility.

Reply:
Further, there seems to be these two options only in the matplotlib case but not for plotly - However the plot should be as identical as possible independent of the plotting backend. How can we make them return the same kind of plot?

Reply:
Two different plots show whether n_forecasts == 1 or n_forecasts >1. See your 1st and 3rd questions for details.

As for the matplotlib and plotly, both have the `plot_nonconformity_scores()`, but only matplotlib has the `plot_interval_width_per_timestep()`. I'll try to code that for plotly now and before the merge of this PR.

Reply:
Just added `plot_interval_width_per_timestep()` for for plotly and modified the `test_plot_conformal_prediction` so the test coverage for that addition is there.

Reply:
It makes more sense to me now, thank you!
From a UI-perspective it's not ideal to have the plot change quite so drastically depending on an external factor, but I think it is OK in this case as this is more of a research/diagnostic plot.